composer.json

```diff
@@ -9,8 +9,9 @@
   "require": {
     "php": ">=7.3",
     "guzzlehttp/guzzle": "^6.3|^7.1",
-    "vasily-kartashov/log4php": "4.1.0",
-    "guzzlehttp/promises": "^1.4"
+    "guzzlehttp/promises": "^1.4",
+    "monolog/monolog": "^2.3",
+    "psr/log": "^1.1"
```

Comment:
Ideally, this library would not require any specific logger implementation, but only `psr/log`. It should be the consumer's duty to supply a compliant logger, with the freedom to choose.